### PR TITLE
Markers and caching some light component parameters.

### PR DESCRIPTION
**Pull request by kecho**
### Purpose of this PR
Adding extra CPU markers for biggest offenders on HDRender Pipeline and core. 

image

---
### Testing status
Tested markers on Razor CPU.

### Comments to reviewers
When looking at the diff of this PR, ensure to disable white space comparison, since when adding markers it forces you to tabulate.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Threading;
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs:4]: I assume this and the one below are leftovers from the experiments?

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs [hunk 2]
```diff
@@ -2185,264 +2187,276 @@ void PreprocessLightData(ref ProcessedLightData processedData, VisibleLight ligh
         // This will go through the list of all visible light and do two main things:
         // - Precompute data that will be reused through the light loop
         // - Discard all lights considered unnecessary (too far away, explicitly discarded by type, ...)
-        int PreprocessVisibleLights(HDCamera hdCamera, CullingResults cullResults, DebugDisplaySettings debugDisplaySettings, in AOVRequestData aovRequest)
+        int PreprocessVisibleLights(CommandBuffer cmd, HDCamera hdCamera, in CullingResults cullResults, DebugDisplaySettings debugDisplaySettings, in AOVRequestData aovRequest)
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs:2190]: I assumed most changes in here are mostly tabs + trivial passing of args to the changed functions, if there is something else to need proper review do let me know

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs (hunk 1 above)
Comment (kecho) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs:4]: yes will fix this

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs (hunk 2 above)
Comment (kecho) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs:2190]: top right settings of the diff, ensure to remove white spaces

**Review by sebastienlagarde — APPROVED** [review]
clean the code reported by Francesco, do Yamato pass (use custom revisoin provide in slack channel for now), and we are good to go

**Review by CertifiedRice — APPROVED** [review]

**Pull request merged**